presto-main/src/main/java/com/facebook/presto/sql/planner/plan/GroupIdNode.java

```diff
@@ -84,6 +90,12 @@ public PlanNode getSource()
         return groupingSets;
     }
 
+    @JsonProperty
+    public Map<Symbol, Symbol> getPassthroughSymbolMap()
```

Comment:
I think we will lose context if we don't say what these mappings actually are. For example, your suggestion would be easy to confuse this with `layout`s in LocalExecutionPlanner or stuff in `UnionNodes`.